docs/pages/reference/machine-id/bound-keypair.mdx

```diff
@@ -0,0 +1,567 @@
+---
+title: Bound Keypair Joining
+description: "Bound Keypair Joining: Reference and admin guide"
+---
+
+Bound Keypair is a new join method designed to provide the best features of
+[dynamic join methods][dynamic] join methods - like AWS, GCP, or Azure - but in
```

Comment:
Should we call them _delegated join methods_ for consistency with https://goteleport.com/docs/reference/join-methods/#secret-vs-delegated